### ISSUE TITLE
Breadcrumbs question

### ISSUE THREAD

**Issue by christiemolloy** [uxd-mk-prototype/uxd-mk-data-plane-ui#13]
@lboehling where should the OpenShift Streams breadcrumb link take you?

<img width="267" alt="Screen Shot 2020-10-27 at 10 28 48 AM" src="https://user-images.githubusercontent.com/20118816/97316135-c9a56c80-183f-11eb-96a1-df134b4bef81.png">

**Comment by lboehling** [comment]
I think this would take you to a page with a list of all the Kafka instances that you have made/have access to. @jgiardino can you clarify?

**Comment by jgiardino** [comment]
Yes, that's correct, you would go back to the Control Plane UI. When we discussed creating this repo, we talked about having a separate `uxd-mk-control-plane-ui` repo, which is where that page would live.
